checkov/terraform/checks/resource/ncp/ServerPublicIP.py

```diff
@@ -0,0 +1,19 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class ServerPublicIP(BaseResourceCheck):
```

Comment:
you can change this to leverage `BaseResourceNegativeValueCheck` like here https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/aws/IAMPolicyAttachedToGroupOrRoles.py 
the `get_inspected_key` would be `server_instance_no` for your check.

Reply:
Thanks. I reflect your advice